helm-chart/amalthea/values.yaml

```diff
@@ -33,6 +33,10 @@ scope:
   # 2. Do not define namespaces at all, in which case amalthea
   #    will only operate in the namespace where the helm chart is deployed.
   # namespaces: ["default"]
+  clusterRoleName: amalthea
+  # name of the cluster role for PV creation
+  # only needs to be set if not deployed clusterwide
+  # the role needs permission to create,get,list,watch PVs
```

Comment:
The info/docs should be above the field they are explaining. Namespaces above is a field and is commented out. So we should move this above the field to avoid confusion. But hopefully we won't need to use this at all on account of the new CSI driver.